contents/docs/libraries/wordpress.md

```diff
@@ -4,60 +4,96 @@ icon: >-
+      posthog.init('<ph_project_api_key>',{api_host:'<ph_client_api_host>', person_profiles: 'identified_only' })
+    </script>	
+	<?php
+}
+endif;
+
+// Hook to add the code right before the closing </head> tag
+add_action('wp_head', 'add_posthog', 999);
+```
+
+After saving your changes or clicking **Update File**, PostHog should begin to autocapture pageviews, clicks, and more.
+
+### Option 2: Edit your theme's header file
+
+If you are using an older version of WordPress, you can edit the `header.php` file directly.
+
+To do this, start by going to your WordPress admin and navigating to **Appearance** -> **Theme Editor**.
+
+Select your theme in the editor drop-down menu to the right and click the `header.php` file in the file column to the right.
     
-    <br />
-5. You should now see the contents of the `header.php` template file in the code editing view. It is recommended that you copy all the text/code and save it somewhere as a back-up.
-6. Find the closing `</head>` in the code editor and paste the PostHog snippet before it (see above image).
-7. Click the 'Update File' button at the bottom. You're good to go - PostHog should now be working on your WordPress website!
+![WordPress Theme Editor](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/tutorials/wordpress/wordpress-header-edit.png)
+
+You should now see the contents of the `header.php` template file in the code editing view. It is recommended that you copy all the text/code and save it somewhere as a back-up.
+
+Find the closing `</head>` in the code editor and paste the PostHog snippet before it (see above image). Finally, click the **Update File** button at the bottom to save your changes. PostHog should begin to autocapture pageviews, clicks, and more.
+
+To confirm PostHog is configured correctly, visit your website and then check if the events from your session appear in PostHog.
+
+> **Notes:** 
+> - Using the Theme Editor is very convenient, but you have to consider the potential drawbacks of having template files writable, which many prefer to disable for security purposes. Also, wrongfully editing a file may cause problems so be sure to perform appropriate backups before attempting this.
+> - If your theme auto-updates, manually editing the `header.php` file may lose your settings. Making a [Child Theme](https://developer.wordpress.org/themes/advanced-topics/child-themes/) is the recommended approach.
+
+### Option 3: Use a plugin
```

Comment:
Given what you say about WordPress.com users, would it be helpful to handle this case first?